sdk/videoanalyzer/Azure.Media.VideoAnalyzer.Edge/README.md

```diff
@@ -26,21 +26,22 @@ this._serviceClient = ServiceClient.CreateFromConnectionString(connectionString)
 
 Install the Video Analyzer client library for .NET with NuGet:
 
-`dotnet add package Azure.Media.VideoAnalyzer.Edge --version 1.0.0-beta.1`
+```bash
+dotnet add package Azure.Media.VideoAnalyzer.Edge --version 1.0.0-beta.1
```

Comment:
But you'll release as "beta.2". For ease - and what we do in our newer betas - is just use:

```bash
dotnet add package Azure.Media.VideoAnalyzer.Edge --prerelease
```

The only time you have to update that is for GA and just drop "--prerelease". If you use the specific version, you're going to have to remember to change it every time.